php/templates/deployment.yaml

```diff
@@ -246,6 +246,12 @@ spec:
         {{- with .Values.extraVolumes }}
         {{- tpl (toYaml .) $root | nindent 8 }}
         {{- end }}
+      {{- with .Values.readinessGates }}
+      readinessGates:
+        {{- range $k, $v := . }}
+        - conditionType: '{{ tpl $v.conditionType $root }}'
+        {{- end }}
```

Comment:
The reason for using range is because of a bug in `toYaml` that causes unnecessary line breaks.
Therefore, we do not use `toYaml` and do the expansion.